.github/workflows/dockerbuild.yml

```diff
@@ -1,16 +1,58 @@
-name: Trigger Docker Build
+name: Docker Build
 
 on:
   push:
-    branches: 
-      - 'master'
+    branches:
+      - master
+  pull_request:
+  release:
+    types:
+      - created
+
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-       - name: Repository Dispatch
-         uses: peter-evans/repository-dispatch@v1
-         with:
-           token: ${{ secrets.DOCKER_REPO_TOKEN }}
-           repository: robertsLando/Zwave2Mqtt-docker
-           event-type: build
+      - uses: actions/checkout@v2
+
+      - name: Just build on PR
+        if: ${{ github.event_name == 'pull_request' }}
+        run: docker build .
```

Comment:
Why this?

Reply:
buildx was slow, figured it'd be nice to know the PR hasn't broken the ability to `docker build` for example changes to the dockerfile

Reply:
and didn't need to mutli-arch build to a reasonable degree of confidence on that